### ISSUE TITLE
Implement updated Address Format

### ISSUE THREAD

**Issue by dignifiedquire** [filecoin-project/go-filecoin#442]
Once #440 is done and the finalized address format is merged into the specs repo, implement the changes into the system.

**Comment by ZenGround0** [comment]
Again https://github.com/filecoin-project/go-filecoin/pull/2186 should close this out and it's tracked in a duplicate issue.
